lib/client.js

```diff
@@ -229,10 +243,28 @@ Client.prototype = {
   /// client.trigger('activation') // activating!
   /// ```
   trigger: function(name, data) {
-    if (!this._messageHandlers[name]) { return false; }
-    this._messageHandlers[name].forEach(function(handler) {
-      handler(data);
-    });
+    if (!this._messageHandlers[name] || repliesPending[name]) { return false; }
```

Comment:
why is one variable local (to the client) and the other global? Could we put them both on the client object?